aiohttp/multipart.py

```diff
@@ -310,13 +311,14 @@ def _read_chunk_from_stream(self, size):
             self._prev_chunk = yield from self._content.read(size)
 
         chunk = yield from self._content.read(size)
-
+        self._content_eof += int(self._content.at_eof())
+        assert self._content_eof < 3, "Reading after EOF"
```

Comment:
Because if 2, tests are failing :)
Seriously, I've send whole request in one chunk:
```
<first_chunk_content><second_chunk>\r\n--boundary--\r\n
```
First `read_chunk(X)` returns `<first_chunk_content>`, buffers all rest of data and counts `self._content_eof` to 1. It also calls `unread_data()`, because boundary is found.
Second `read_chunk(Y)` first reads new chunk (got boundary and \r\n), increments `self._content_eof` to 2. Then it returns `<second_chunk>`, pushes boundary again and marks `BodyPartReader` with `at_eof=True`.

I can't come up with expressing this in better way, however magic numbers are bad. Any ideas?